frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/components/SelectSignature.js

```diff
@@ -67,7 +67,6 @@ function SelectSignature({ t, config, onSelect, formData = {}, errors }) {
         newobj[key] && !isEqual(formData[key], newobj[key]) && setValue(key, newobj[key], Object.keys(newobj[key]));
       });
       localStorage.removeItem("signStatus");
-      localStorage.removeItem("esignProcess");
```

Comment:
why is this removed ? I don't remember this being removed in your PR to develop @vaibhavct